specutils/fitting/fitmodels.py

```diff
@@ -0,0 +1,507 @@
+from __future__ import division
+
+import functools
+import operator
+
+import numpy as np
+import astropy.units as u
+
+from ..spectra.spectrum1d import Spectrum1D
+from ..manipulation.utils import excise_regions
+from astropy.modeling import fitting, Model, models
+
+
+__all__ = ['fit_lines']
+
+
+def fit_lines(spectrum, model, fitter=fitting.SimplexLSQFitter(),
+              exclude_regions=None, weights=None, window=None):
+    """
+    Fit the input models (initial conditions) to the spectrum.  Output will be
+    the same models with the parameters set based on the fitting.
+
+    spectrum, models -> models
+
+    Parameters
+    ----------
+    spectrum : Spectrum1D
+        The spectrum object overwhich the equivalent width will be calculated.
+
+    model: `~astropy.modeling.Model` or list of `~astropy.modeling.Model`
+        The model or list of models that contain the initial guess.
+
+    exclude_regions : list of `~specutils.SpectralRegion`
+        List of regions to exclude in the fitting.
+
+    weights : list  (NOT IMPLEMENTED YET)
+        List of weights to define importance of fitting regions.
+
```

Comment:
``window`` is missing in this docstring...

Reply:
Added, but that brings up another point. `window` should probably be based on SpectralRegion now that we have it.